File: VB/WindowsFormsApplication741/MyPopupToolWindowLeftRightSlideAnimationProvider.vb
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports DevExpress.Utils.Win

Namespace WindowsFormsApplication741
    Public Class MyPopupToolWindowLeftRightSlideAnimationProvider
        Inherits PopupToolWindowLeftRightSlideAnimationProvider

        Public Sub New(ByVal info As IPopupToolWindowAnimationSupports)
            MyBase.New(info)
        End Sub
        Public Overrides Function CalcTargetFormSize() As Size
            Return Info.FormSize
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication741/MyPopupToolWindowLeftRightSlideAnimationProvider.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using DevExpress.Utils.Win;

namespace WindowsFormsApplication741 {
    public class MyPopupToolWindowLeftRightSlideAnimationProvider : PopupToolWindowLeftRightSlideAnimationProvider {
        public MyPopupToolWindowLeftRightSlideAnimationProvider(IPopupToolWindowAnimationSupports info)
            : base(info) {
        }
        public override Size CalcTargetFormSize() {
            return Info.FormSize;
        }
    }
}
